framework/test/unit/modules/interoperability/method.spec.ts

```diff
@@ -456,6 +457,36 @@ describe('Sample Method', () => {
 		});
 	});
 
+	describe('getMinReturnFeePerByte', () => {
+		const newChainID = Buffer.from('00000000', 'hex');
+		beforeEach(() => {
+			jest.spyOn(channelStoreMock, 'has').mockResolvedValue(true);
+		});
+
+		it('should assign chainID as mainchain ID if ownchain ID is not mainchain ID and chainAccount not found', async () => {
+			await sampleInteroperabilityMethod.getMinReturnFeePerByte(methodContext, newChainID);
+			expect(channelStoreMock.get).toHaveBeenCalledWith(
+				expect.anything(),
+				getMainchainID(newChainID),
+			);
+		});
+
+		it('should process with input chainID', async () => {
```

Comment:
```suggestion
		it('should process with input with valid chainID', async () => {
```

Reply:
Valid can mean either mainchainID or the new chain ID. It is more specific to say it's the same ID as the input ID. Also for consistency it is using the same description as the one used in the above test suite for `getMessageFeeTokenID`.

Reply:
I meant to say to distinguish from the other scenarios, we can add something, like `should successfully execute for the valid input chainID` but yeah just my opinion so its optional